src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -38,43 +38,43 @@ Above setting increases the TTL to 60 seconds.
 
 The cache plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/cache/).
 
-## Additional proxies
+## Additional forwards (formerly known as proxy)
```

Comment:
```suggestion
## Additional forwards (formerly known as proxy) {#additional-forwards}
```

This will make the ID for this headline `additional-forwards`, which allows for some longer living link targets once we change that headline again.